portal-frontend/src/app/features/applications/application-details/nfu-details/nfu-details.component.html

```diff
@@ -72,32 +72,43 @@ <h4>Soil and Fill Components</h4>
       ></app-no-data>
     </div>
 
-    <div class="grid-1"></div>
-    <div class="subheading2 grid-double">Fill to be Placed</div>
-
-    <div class="subheading2 grid-1">Volume</div>
-    <div class="grid-double">
-      {{ applicationSubmission.nfuFillVolume }}
-      <span *ngIf="applicationSubmission.nfuFillVolume !== null">m<sup>3</sup></span>
-      <app-no-data [showRequired]="showErrors" *ngIf="applicationSubmission.nfuFillVolume === null"></app-no-data>
-    </div>
-    <div class="subheading2 grid-1">Area=</div>
-    <div class="grid-double">
-      {{ applicationSubmission.nfuTotalFillArea }}
-      <span *ngIf="applicationSubmission.nfuTotalFillArea !== null">m<sup>2</sup></span>
-      <app-no-data [showRequired]="showErrors" *ngIf="applicationSubmission.nfuTotalFillArea === null"></app-no-data>
-    </div>
-    <div class="subheading2 grid-1">Maximum Depth</div>
-    <div class="grid-double">
-      {{ applicationSubmission.nfuMaxFillDepth }}
-      <span *ngIf="applicationSubmission.nfuMaxFillDepth !== null">m</span>
-      <app-no-data [showRequired]="showErrors" *ngIf="applicationSubmission.nfuMaxFillDepth === null"></app-no-data>
-    </div>
-    <div class="subheading2 grid-1">Average Depth</div>
-    <div class="grid-double">
-      {{ applicationSubmission.nfuAverageFillDepth }}
-      <span *ngIf="applicationSubmission.nfuAverageFillDepth !== null">m</span>
-      <app-no-data [showRequired]="showErrors" *ngIf="applicationSubmission.nfuAverageFillDepth === null"></app-no-data>
+    <div class="full-width soil-table">
+      <div></div>
```

Comment:
Same as [previous comment](https://github.com/bcgov/alcs/pull/1255/files#r1437797145)

Reply:
yes, the first column header is empty